### ISSUE TITLE
Support option to generate a RTL or a LTR version only, rather than combined

### ISSUE THREAD

**Issue by bencripps** [vkalinichev/postcss-rtl#27]
It would be great if you could pass in some configuration which resulted in either LTR *or* RTL output -- rather than combined. 

Have you thought about adding this feature?

**Comment by vkalinichev** [comment]
Not sure it's necessary. You can do it directly via `rtlcss` used by this project

**Comment by TsuryKnox** [comment]
@vkalinichev how is this possible using rtlcss?

**Comment by bencripps** [comment]
This postcss plugin has some nice configuration that isn't (yet) supported by rtlcss -- the feature I'm looking for in particular is the skipping of classes in which `[dir]` is already part of the selector.

For example, using rtlcss, the following CSS:

```cs
[dir="rtl"] .thing { left: 10px }
```

would be rewritten as  

```cs
[dir="rtl"] .thing { right: 10px }
```
which isn't desirable. 

Using postcssrtl, this CSS go unmodifed.

**Comment by vkalinichev** [comment]
Implemented by `onlyDirection` option in `1.2.1` release.

**Comment by bencripps** [comment]
Thank you for implementing this! However, when trying to use it with the following code:

```
     p.use(require('postcss-rtl')({ onlyDirection: 'rtl' }));
```

I still see a bunch of [dir="ltr"] throughout the generated CSS -- is that the outcome I should be expecting?
